download/sfml/2.5.1/index.php

```diff
@@ -23,7 +23,8 @@ function download_link($compiler, $arch, $size, $link)
                 compile and run a 32-bit program on a 64-bit Windows. So you'll most likely want to target 32-bit platforms, to have the largest possible audience.
                 Choose 64-bit packages only if you have good reasons.
                 <div class="important">
-                    <strong>The compiler versions have to match 100%!</strong><br>
+                    <strong>Unless you are using newer versions of Visual Studio the compiler versions have to match 100%!</strong><br>
+                    In case you are using Visual Studio 2015 or newer you can go ahead and use the latest Visual C++ 15 (2017) version which is still compatible with newer versions of the toolchain.<br>
```

Comment:
> and use the latest Visual C++ 15 (2017) version

This part is slightly confusing, and the sentence could also use some commas.
Maybe something like:

> In case you are using Visual Studio 2015 or newer, you can go ahead and use the SFML version compiled for Visual C++ 15 (VS 2017), which is still compatible with newer versions of the toolchain.<br>

Reply:
Done.